videointelligence/google/cloud/videointelligence_v1/gapic/video_intelligence_service_client.py

```diff
@@ -187,7 +172,19 @@ def __init__(
         # transport methods, wrapped with `wrap_method` to add retry,
         # timeout, and the like.
         self._inner_api_calls = {}
+        
+    def channel(self):
+        return self.transport.channel
 
+    def get_operation(self, name=None):
+        return self.transport.get_operation(name)
+        
+    def delete_operation(self, name=None):
+        return self.transport.delete_operation(name)
+        
+    def cancel_operation(self, name=None):
+        return self.transport.cancel_operation(name)
```

Comment:
Change to generated code. These three methods appear to be added only FBO the system test below.

Reply:
The API supports these methods so they should be in the generated code. We need to have integration tests for all supported methods. Is there an internal way of updating the generated code?

Reply:
Hi @mkudejim, thank you for your patience.

I believe these methods aren't in the client because they aren't in the protos. (If the methods have been added recently, please sync the protos to third party)
[videointelligence_v1.proto](https://github.com/googleapis/googleapis/blob/4896ff3515b8cb3ba0e02abcd44af2a67f14c28d/google/cloud/videointelligence/v1/video_intelligence.proto)

The Container API uses operations and their fully generated client has the desired methods. Please reference it as you make changes.

https://github.com/googleapis/googleapis/blob/b4c73face84fefb967ef6c72f0eae64faf67895f/google/container/v1beta1/cluster_service.proto#L209-L239

generated methods in transport and client:
https://github.com/googleapis/google-cloud-python/blob/f657b033b1b5ff7d5e94e6d1d0912484c2e5adb2/container/google/cloud/container_v1/gapic/transports/cluster_manager_grpc_transport.py#L294-L331

https://github.com/googleapis/google-cloud-python/blob/f657b033b1b5ff7d5e94e6d1d0912484c2e5adb2/container/google/cloud/container_v1/gapic/cluster_manager_client.py#L1531-L1794